website/www/site/content/en/roadmap/portability.md

```diff
@@ -22,11 +22,11 @@ limitations under the License.
 ## Overview
 
 Interoperability between SDKs and runners is a key aspect of Apache
-Beam. So far, however, the reality is that most runners support the
-Java SDK only, because each SDK-runner combination requires non-trivial
-work on both sides. All runners are also currently written in Java,
+Beam. Previously, the reality was that most runners supported the
+Java SDK only, because each SDK-runner combination required non-trivial
+work on both sides. Most runners are also currently written in Java,
 which makes support of non-Java SDKs far more expensive. The
-_portability framework_ aims to rectify this situation and provide
+_portability framework_ aims to rectified this situation and provided
```

Comment:
did you mean this sentence to be in past tense? "aims to rectified"

Reply:
Fixed.